documentation/how-to/generate-service-account.md

```diff
@@ -2,11 +2,30 @@
 title: Generate a Service Account
 ---
 
-1. Go to the `Namespaces` tab.
+<!-- overview -->
 
-2. Click `Service Accounts`, then Cick `New Service Account`.
+This guide explains how to generate a service account.
 
-3. Select the `GatewayConfig.Publish` permission for the Service Account and click `Share`. A new credential will be created - make a note of the `ID` and `Secret`.
+{Optional: Specify when and why your user might want to perform the task.}
+A service account can be used instead of IDIR to login to the `gwa` command line interface (CLI).
+This is useful for intergrating with CI/CD pipelines.
+Ask aidan about other benefits
+
+## Before you begin
+
+Before you begin, ensure you:
+
+- [Create a Namespace](/resources/gwa-commands.md#namespacecreate)
```

Comment:
should this be called out further? like a 'stop you cannot proceed without completing the steps here @Elson9

Reply:
What James has included is following the [template ](https://github.com/bcgov/aps-infra-platform/blob/test/documentation/how-to/!how-to-template.md) and I think is adequate.